HW8/src/ArrayValueCalculator.java

```diff
@@ -0,0 +1,46 @@
+import exceptions.ArrayDataException;
+import exceptions.ArraySizeException;
+
+public class ArrayValueCalculator {
+
+    public static void main(String[] args) throws ArrayDataException, ArraySizeException {
```

Comment:
It is a bad idea to throw exception outer of your scope (you do not control what is happening outer method main). So, please handle those exceptions in main method.

Reply:
Was handled in main